File: 0023.py
def main():
    days = ['Monday','Tuesday','Wednesday','Thursday','Friday','Saturday','Sunday']
    months = [31,28,31,30,31,30,31,30,31,30,31]
    d,m = list(map(int,input().split()))
    x = d+sum(months[:m-1])+2
    print(days[x%7])
    
if __name__ == "__main__":
    main()

File: c++/Programming/00/0023.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    int D, M;
    cin >> D >> M;
    string week[7] = {"Thursday", "Friday", "Saturday", "Sunday", "Monday", "Tuesday", "Wednesday"};
    int month[13] = {0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
    int ans = 0;
    for(int i = 1; i < M; i++){
        ans += month[i];
    }
    ans += D - 1;
    cout << week[ans % 7];
}